src/main/zapHomeFiles/hud/tools/activeScan.js

```diff
@@ -87,12 +87,43 @@ var ActiveScan = (function() {
 			.catch(utils.errorHandler);
 	}
 
-	function startActiveScan(tabId, domain) {
+	function startActiveScanDomain(tabId, domain) {
 		utils.getUpgradedDomain(domain)
 			.then(upgradedDomain => {
-				return utils.zapApiCall("/ascan/action/scan/?url=" + upgradedDomain)
-			}).
-			then(response => {
+				startActiveScan(tabId, upgradedDomain, "true", "GET", "");
+			})
+			.catch(utils.errorHandler);
+	}
+	
+	function showReqDialog(tabId, uri, method, body) {
```

Comment:
what is the purpose of this function? I noticed it wasn't called anywhere.

Reply:
Ah yeah, I did expetiment with adding a dialog to confirm that an ascan should be started, but in the end didnt use it. Will delete.